kedro/framework/cli/starters.py

```diff
@@ -851,9 +851,12 @@ def _create_project(template_path: str, cookiecutter_args: dict[str, Any]):
         f"\nYour project '{project_name}' has been created in the directory \n{result_path}\n"
     )
 
-    # we can use starters without tools:
+    # End here if a starter was used
+    if template_path != TEMPLATE_PATH:
+        return
+
     if tools is not None:
-        if tools == "['None']":  # TODO: This should be a list
```

Comment:
Removed the comment as the string to list conversion is done later in the execution timeline, and the final item stored in `pyproject.toml` is indeed a list object. This conversion cannot happen before the cookiecutter generation, as `cookiecutter` does not store lists but rather parses them as a selection of options, only storing one of the values within the list.